docs/pages/versions/unversioned/sdk/cellular.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Cellular
+---
```

Comment:
this is an awesome docs page. great job 👏

Reply:
Thank u!!